docs/usage/configuration-options.md

```diff
@@ -668,6 +668,8 @@ This is caused by a race condition in GitLab's Merge Request API - [read the cor
 
 Configuration added here applies for all Go-related updates, however currently the only supported package manager for Go is the native Go Modules (the `gomod` manager).
 
+For self-hosted users, `GOPROXY`, `GONOPROXY` and `GOPRIVATE` environment variables are supported, though `direct` and `off` keywords will cause fallback to default Renovate fetching strategy (i.e. directly from GitHub, etc).
```

Comment:
```suggestion
For self-hosted users, `GOPROXY`, `GONOPROXY` and `GOPRIVATE` environment variables are supported.
But when you use the `direct` and/or `off` keywords Renovate will fallback to the default fetching strategy (i.e. directly from GitHub, etc).
```

How's this? Is it when you use `direct` _AND_ `off keyword that you get the fallback, or is it also the case that if you use either one of those you get the fallback?

Please check that the suggestion has the correct description of the backend behavior. 😉

Reply:
Thanks for checking this out!

Renovate's interpretation of `direct` keyword is same, while `off` doesn't make sense for us (we **must** obtain deps somewhere). So, "Renovate will fallback to *its own* default fetching strategy".

Reply:
Thank you for fixing the text. 😄